Document X-Forwarded-For capabilities within access-control networks (#1794)
authelia/authelia: docs/configuration/access-control.md

policies when requests originate from different networks. This list can contain both literal definitions of networks
and [network aliases](#network-aliases).

> Network addresses specified will be matched against the IP address of the request, as well as addresses
> contained within the X-Forwarded-For HTTP header. If using Authelia with a reverse proxy, additional configuration
> may be required on the reverse proxy to ensure these headers are present and correct.

Reviewer: ```suggestion
Network addresses specified will be matched against the first IP in the X-Forwarded-Header, and if there is none it will fall back to the IP address of the request. If using Authelia with a reverse proxy, additional configuration
may be required on the reverse proxy to ensure these headers are present and correct.
```

Reviewer: Maybe something like this? X-Forwarded-For takes precedence over anything else.
